configs/pontoon.toml

```diff
@@ -115,6 +115,16 @@ locales = [
     "zu",
 ]
 
+[[paths]]
```

Comment:
@peiying2 this is the list of supported locales for this page as per langchecker: https://github.com/mozilla-l10n/langchecker/blob/master/app/config/sources.inc.php#L258-L268

Is this restriction still wanted / needed, or would it be better to open this up for all locales?

I notice the nightly /firstrun page is supported more broadly in comparison: https://github.com/mozilla-l10n/langchecker/blob/master/app/config/sources.inc.php#L255-L257

Reply:
Both files were created 4-5 years ago, with nightly/whatsnew.lang updated earlier this year. Unless it is tied to the actual released locales to nightly, I am ok to open it to more locales.

@flodolo need your help with the background info.

Reply:
@alexgibson we can open it to all locales. I re-read the setting in langchecker, it is open to all Firefox locales, but communities needs to opt-in to make it available for localization. nightly_firstrun.lang is an older file and it was open to all locales. Not sure what drove the change.

Reply:
As far as I can tell, the locales available in Nightly are a superset of the locales available in mozilla.org

The limitation comes from a time where we had limited locales on Nightly (we had Aurora, which was the focus for localization), but it doesn't make sense anymore these days, for both whatsnew and firstrun.